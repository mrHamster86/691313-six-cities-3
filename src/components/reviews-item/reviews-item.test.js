import React from 'react';
import renderer from 'react-test-renderer';
import ReviewsItem from './reviews-item';

const user = `Max`;
const avatar = `img/avatar.jpg`;
const rating = 4;
const date = `April 2020`;
const text = `Comment`;

it(`ReviewsItem components renders`, () => {
  const tree = renderer.create(
      <ReviewsItem
        user={user}
        avatar={avatar}
        rating={rating}
        date={date}
        text={text}
      />).toJSON();
  expect(tree).toMatchSnapshot();
});
