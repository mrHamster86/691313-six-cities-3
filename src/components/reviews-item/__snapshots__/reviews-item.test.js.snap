// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsItem components renders 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Max"
        className="reviews__avatar user__avatar"
        height="54"
        src="img/avatar.jpg"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Max
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "80%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      Comment
    </p>
    <time
      className="reviews__time"
      dateTime="April 2020"
    >
      Apr 2020
    </time>
  </div>
</li>
`;
